server/connection_manager.py

```diff
@@ -0,0 +1,130 @@
+from dataclasses import dataclass
+from typing import Literal, TypedDict
+from uuid import UUID
+
+from server.client import Client
+from server.errors import RoomNotFoundError
+from server.events import EventResponse, ReplaceData
+
+
+@dataclass
+class RoomData:
+    """A dataclass for data about a specific room."""
+
+    owner_id: UUID
+    clients: set[Client]
+    code: str
+
+
+class ActiveRooms(TypedDict):
+    """A data structure for active rooms."""
+
+    name: str
+    data: RoomData
```

Comment:
Can you replace all of this with:
```py
class RoomData(TypedDict):
    """A dataclass for data about a specific room."""

    owner_id: UUID
    clients: set[Client]
    code: str


ActiveRooms: TypeAlias = dict[str, RoomData]
```
This should fix the type issues. Remove the dataclass import and add `TypeAlias` to the typing import

Reply:
Fixed in 197e3aaf67a49d8ad35c73a61f679dbcf8807fc5.